src/Description/Description.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./Description.css";
+import { usePlayerState } from "../playerState";
+
+const Description = () => {
+  const player = usePlayerState();
+  // starting string, can be changed according to player's last location
+  let description = "You are in a grassy field. ";
+
+  // array of arrays, first element of each subarray is a query into the player's stats,
+  // second element is a bit of info to add to the player's description
+  // ex: details[0] takes in the player's hp and maxhp and if the hp is less then half
+  // of the maxhp, gives back a "you are hurt" message.
+  const details = [
+    [({ hp, maxhp }) => hp < maxhp / 2, "You are hurt. "],
+    [({ attack }) => attack < 3, "You feel weak. "],
+    [({ defense }) => defense < 3, "You feel defenseless. "],
+    [({ name }) => !name, "You feel a loss of identity. "],
+  ];
```

Comment:
You can tell you used to make games, these are literally the little things I wouldn't have thought about. 
Love the "You feel a loss of identity" 👍

Reply:
Yeah, I'd imagine we'd add to this as time goes on with more tidbits of information